### ISSUE TITLE
Two szenarios with overlapping cells

### ISSUE THREAD

**Issue by nicosurf** [kuleszdl/SIFEI#135]
When first creating one szenario with a certain set of cells and then creating a second set of cells can cause problems when the intersection of these two sets is not empty. 

Then all cells from the first scenario are taken plus the newly defined ones.
A relabeling is possible thouh (An input cell in scenario A can be an output in scenario B)

**Comment by kuleszdl** [comment]
Yes, but actually you can also remove cells from a scenario (although you can't add them), so the marked cells are more or less what is the default for creating new scenarios. In the studies, quite many users thought that when creating a second scenario you have to do the cell-markings again, so there is definitely room for improvement here.

Can you provide more details (as a test case) for the mentioned issue with overlapping? What is the expected behavior and what happens instead?

**Comment by nicosurf** [comment]
Ok now with your description it makes sense. 
Marked cells will stay marked even though the creation of the first scenario is done. 
Thats why the will also appear in the new scenario.

Didn't think that this is the expected/desired behavior. Then not a bug but maybe a Usability/knowledge problem

**Comment by friederschueler** [comment]
Please mark as SIFEI bug. Is this situation fixed? I think the expected behavior is to unmark all cells when a new scenario is created // saved.

**Comment by kuleszdl** [comment]
No, actually it makes sense to leave the markings intact because you could have multiple scenarios for the same group of cells.
However, if scenarios are added twice this is definitely a bug.
